sparse/_compressed/compressed.py

```diff
@@ -25,7 +25,7 @@ def _from_coo(x, compressed_axes=None):
         if compressed_axes is not None:
             raise ValueError('no axes to compress for 1d array')
         return (
-            x.data, x.coords[0], []), x.shape, None, None, None, None, None, x.fill_value
+            x.data, x.coords[0], np.array([])), x.shape, None, None, None, None, None, x.fill_value
```

Comment:
This will make the dtype `float64`. In general, I like keeping things that aren't really arrays as tuples of integers or similar.